src/renderer/components/Text.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react'
+import { PropsWithChildren } from 'react'
+import Typography, { TypographyProps } from '@mui/material/Typography'
+
+type BaseProps = PropsWithChildren &
```

Comment:
not sure if we should extend MUI's props here because then the exported components are more closely tied to MUI. would be selective and only specify the props that are most relevant for our use cases and specify them without referencing MUI (is possible). Example:

```tsx
type BaseProps = PropsWithChildren<{
  align?: Extract<
    CSSProperties['textAlign'],
    'inherit' | 'left' | 'center' | 'right' | 'justify'
  >
  bold?: boolean
  className?: string
  id?: string
  italic?: boolean
  style?: CSSProperties
  underlined?: boolean
}>

export function TitleText({
  align,
  bold,
  children,
  className,
  style,
}: BaseProps) {
  return (
    <Typography
      variant="h1"
      align={align}
      style={style}
      fontWeight={bold ? 'bold' : undefined}
      className={className}
    >
      {children}
    </Typography>
  )
}
```

not exactly sure what kind of props should be exposed for end usage. the ones listed in the example are probably sufficient for now, but could imagine wanting other things like data attributes or aria attributes in the future